src/components/ha-mount-picker.ts

```diff
@@ -137,6 +138,7 @@ class HaMountPicker extends LitElement {
     try {
       if (isComponentLoaded(this.hass, "hassio")) {
         this._mounts = await fetchSupervisorMounts(this.hass);
+        this.value = this._mounts.default_backup_mount || __BACKUP_DATA_DISK__;
```

Comment:
By setting this, we are showing whats in use now when opening using the selector.

Reply:
Add a guard to only do this if usage is backup

Reply:
Huh? usage would always be backup here.
If its undefined or null, it means the same, /bckup is in use

Reply:
This is an input element for picking mounts, it also allows media, but is not used now. 

But if the user picked something else (like it is used in a service call, or blueprint, etc) the value could be set already and we now override that value with the default, which is wrong.
You can also move this logic to the backup location dialog.

Reply:
No? its a `backup_location` selector, that always are backup

Reply:
`default_backup_mount` from SU API is always also backup mount.
Everything here is backup only

Reply:
Oh.... F, this is in mount picker...
Yeah...